extensions/applicationinsights-clickanalytics-js/src/Interfaces/Datamodel.ts

```diff
@@ -38,16 +38,21 @@ export interface IClickAnalyticsConfiguration {
      * value will be ovverriden if the element has the data-*-bhvr attribute present.
      */
     defaultRightClickBhvr?: string | number;
+    /**
+     * Flag to drop events that donot have custom event names, no parentId and no data in content (basically no useful click data).
+     * Default will be false
+     */
+    disableUndefinedEventsTracking?: boolean;
```

Comment:
I am open to a better name, if you think its not very intuitive

Reply:
It does seem a bit long, but it is descriptive -- not sure about describing them as undefined though -- as we are talking about events here are some suggestions
- trackUndefined -- default to true
- ignoreUndefined -- default to false
- ignoreInvalid -- default to false
- dropInvalid -- default to false
- skipInvalid -- default to false

Reply:
thanks for your suggestion. dropInvalidEvents looks better I guess